appengine/standard/ndb/schema_update/add_entities_handler.py

```diff
@@ -0,0 +1,32 @@
+import os
+import time
+
+import jinja2
+import webapp2
+
+import update_schema
```

Comment:
done. thanks for the tip - moved add_entity function to the add_entities_handler module